sdk/azcore/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Features Added
 * Added header `WWW-Authenticate` to the default allow-list of headers for logging.
+* Added a pipeline policy that enables the retrieval of HTTP responses from API calls.
+  * Added `policy.IncludeResponse` to enable the policy at the API level (off by default).
+  * Added `runtime.ResponseFromContext` to retreive the HTTP response from the context when enabled.
```

Comment:
Are we ok with having the functionality split across two packages?  I feel like `ResponseFromContext` makes sense in the `runtime` package.

Reply:
I think `IncludeResponse` could go in `runtime` as well because it configures runtime behavior. The fact that an unexported policy implements the behavior is an implementation detail.

Reply:
I think you're right.  This also makes me thing that the other WithXxx funcs should go in `runtime` too.  What do you think?